BUILD-sbi

```diff
@@ -0,0 +1,29 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@rules_rust//rust:defs.bzl", "rust_clippy", "rust_library", "rustfmt_test")
+
+rust_library(
+    name = "sbi-rs",
+    srcs = glob(["src/**/*.rs"]),
+    crate_root = "src/sbi.rs",
+    proc_macro_deps = ["@salus-index//:enum_dispatch"],
+    target_compatible_with = [
+        "@platforms//cpu:riscv64",
+        "@platforms//os:none",
+    ],
+    deps = [
+        "@salus-index//:arrayvec",
+        "@salus-index//:flagset",
+        "@salus-index//:static_assertions",
```

Comment:
I assume sbi-rs is in its own module so it can be used elsewhere too? At some point it would probably be good to set up sbi-rs with its own WORKSPACE and crates-index and repository defs; but that's a hassle and not something we need to do now.

Reply:
fixed